app/controllers/companies_controller.rb

```diff
@@ -1,4 +1,19 @@
 class CompaniesController < ApplicationController
+  def dashboard
+    @sorted_companies = find_top_three_companies
+    @jobs_count = Job.group(:city).count("id")
+  end
+
+  def find_top_three_companies
```

Comment:
i like this method, but does it belong in your controller? what's a more appropriate place for it?